src/transformers/models/auto/configuration_auto.py

```diff
@@ -186,7 +186,7 @@
         ("pegasus", PegasusConfig),
         ("marian", MarianConfig),
         ("mbart", MBartConfig),
-        ("megatron_bert", MegatronBertConfig),
+        ("megatron-bert", MegatronBertConfig),
```

Comment:
It looks like a typo since the model type in the config is `"megatron-bert"`. Can change to put both keys if somehow the two handed up being present.

Reply:
It looks like a typo indeed